components/Graphs/most-used-languages-graph.tsx

```diff
@@ -0,0 +1,132 @@
+import { BsFillCircleFill } from "react-icons/bs";
+import Button from "components/atoms/Button/button";
+import Card from "components/atoms/Card/card";
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+} from "components/atoms/Dropdown/dropdown";
+import PeopleIcon from "img/icons/people.svg";
+import ChevronDownIcon from "img/chevron-down.svg";
+import SVGIcon from "components/atoms/SVGIcon/svg-icon";
+import Icon from "components/atoms/Icon/icon";
+import Text from "components/atoms/Typography/text";
+
+export interface MostUsedLanguagesGraphProps {
+  data: {
```

Comment:
I don't know what the shape of the data is yet, but this can be updated once I've created the API endpoint.